package.json

```
@@ -1,179 +1,19 @@
-    "jsdoc-to-markdown": "^6.0.1",
-    "npm-run-all": "^4.1.5",
-    "null-loader": "^4.0.0",
-    "postinstall-postinstall": "^2.1.0",
-    "prettier": "^2.0.1",
-    "raf": "^3.4.1",
-    "react-dev-utils": "^11.0.0",
-    "static-site-generator-webpack-plugin": "^3.4.2",
-    "url-loader": "^4.0.0",
-    "webpack": "^4.42.0",
-    "webpack-cli": "^4.4.0",
-    "webpack-dev-server": "^3.10.3",
-    "webpack-manifest-plugin": "^3.0.0",
-    "webpack-node-externals": "^2.5.2"
-  },
-  "greenkeeper": {
-    "ignore": [
-      "history",
-      "react-dev-utils",
-      "react-router",
-      "tmp"
-    ]
+    "flow-mono-cli": "^1.5.3",
+    "lerna": "^4.0.0"
   },
   "scripts": {
-    "prettify": "prettier --write '**/*.{js,md,yml}'",
-    "check-pretty": "prettier --list-different '**/*.{js,md,yml}'",
-    "start": "./scripts/devServer.sh",
-    "build": "run-p build:* && cp -r ./build ./bin/site-template && chmod +x ./bin/sourcecred.js",
-    "build:frontend": "NODE_ENV=production webpack --config config/webpack.config.web.js",
-    "build:backend": "NODE_ENV=development webpack --config config/webpack.config.backend.js",
-    "build:api": "NODE_ENV=production webpack --config config/webpack.config.api.js",
-    "test": "node ./config/test.js",
-    "createLedger": "babel-node ./scripts/createLedgerFromLegacyProject.js",
-    "unit": "BABEL_ENV=test NODE_ENV=test jest --env=jsdom",
-    "sharness": "make -sC ./sharness prove PROVE_OPTS=-f TEST_OPTS='--chain-lint'",
-    "sharness-full": "make -sC ./sharness prove PROVE_OPTS=-vf TEST_OPTS='-v --chain-lint --long'",
-    "coverage": "yarn run unit --coverage",
-    "flow": "flow",
-    "lint": "eslint src config --max-warnings 0",
-    "prepublishOnly": "yarn build",
-    "docs": "node ./scripts/build-docs.js"
-  },
-  "engines": {
-    "node": ">=12"
-  },
-  "license": "(MIT OR Apache-2.0)",
-  "jest": {
-    "collectCoverageFrom": [
-      "src/**/*.{js,jsx,mjs}"
-    ],
-    "setupFiles": [
-      "<rootDir>/config/polyfills.js",
-      "<rootDir>/config/jest/setupJest.js"
-    ],
-    "testMatch": [
-      "<rootDir>/src/**/__tests__/**/*.{js,jsx,mjs}",
-      "<rootDir>/src/**/?(*.)(spec|test).{js,jsx,mjs}"
-    ],
-    "testEnvironment": "node",
-    "testURL": "http://localhost",
-    "transform": {
-      "^.+\\.(js|jsx|mjs)$": "<rootDir>/node_modules/babel-jest",
-      "^.+\\.css$": "<rootDir>/config/jest/cssTransform.js",
-      "^(?!.*\\.(js|jsx|mjs|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
-    },
-    "transformIgnorePatterns": [
-      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs)$"
-    ],
-    "moduleNameMapper": {
-      "^react-native$": "react-native-web"
-    },
-    "moduleFileExtensions": [
-      "web.js",
-      "js",
-      "json",
-      "web.jsx",
-      "jsx",
-      "node",
-      "mjs"
-    ]
+    "test": "lerna run test --",
+    "build": "lerna run build",
+    "install": "lerna bootstrap ; ./scripts/flow-mono.sh",
+    "start": "lerna run start"
```

Lerna doesn't print package process outputs until the jobs finish. These flags will allow packages to print to the console in real time.

```suggestion
    "test": "lerna run test --stream --",
    "build": "lerna run build --stream",
    "install": "lerna bootstrap ; ./scripts/flow-mono.sh",
    "start": "lerna run start --stream"
```

---

That is awesome thanks for that!